### PR TITLE
docs: fix quickstart with some breaking changes

### PR DESCRIPTION
**Pull request by h4ck3rk3y**
## Description:
Had missed service_name from 0.71.0 in one place
Args serialization changed in 0.72.0

## Is this change user facing?
YES

### CODE REVIEW

File: docs/docs/quickstart.md [hunk 1]
```diff
@@ -820,7 +820,7 @@ def run(plan, args):
     )
 
     # Insert data
-    if args != None:
+    if "data" in args:
```
Comment (gbouv) [inline docs/docs/quickstart.md]: Technically we could also check if the dictionary is empty with a `len()` such that we're more future proof here, but I think that's find. If the structure of the json changes, this code will probably break somewhere else anyway

File: docs/versioned_docs/version-0.72.0/quickstart.md [hunk 2]
```diff
@@ -840,7 +840,7 @@ def insert_data(plan, data):
 Now clean and run, only this time with extra args to `kurtosis run`:
 
 ```bash
-kurtosis clean -a && kurtosis run --enclave quickstart . '[{"first_name":"Kevin", "last_name": "Bacon"}, {"first_name":"Steve", "last_name":"Buscemi"}]'
+kurtosis clean -a && kurtosis run --enclave quickstart . '{"data": [{"first_name":"Kevin", "last_name": "Bacon"}, {"first_name":"Steve", "last_name":"Buscemi"}]}'
```
Comment (gbouv) [inline docs/versioned_docs/version-0.72.0/quickstart.md]: Any chance we can change `data` to something more on topic here? With `data`, it feels a bit like a hack, whereas if we had something like `{persons: [kevin, steve, etc.]}` mayyybe that'd be nicer? 
Deferring to you here

**Review by gbouv — APPROVED** [review]
One suggestion, but don't block on it if you disagree

File: docs/versioned_docs/version-0.72.0/quickstart.md (hunk 2 above)
Comment (h4ck3rk3y) [reply docs/versioned_docs/version-0.72.0/quickstart.md]: point i can just call it `actors` I suppose 👍

**Pull request merged**